oval_graph/_xml_parser_oval_scan_definitions.py

```diff
@@ -28,12 +28,23 @@ def _get_negate_status(self, node):
             negate_status = str_to_bool[node.get('negate')]
         return negate_status
 
+    def _get_result(self, negate_status, tree):
+        result = tree.get('result')
+        reverse_negate_value = {
+            'true': 'false',
+            'false': 'true',
+        }
+        if negate_status and result in ('true', 'false'):
+            result = reverse_negate_value[result]
+        return result
```

Comment:
I believe you can improve this function by adding a comment. This function does something "unexpected" for a person that sees it for the first time. If the person sees that, they might think "WTH". A comment will help them to understand this quickly. Comment can be similar to the PR description of this PR.